packages/backend-core/src/db/instrumentation.ts

```diff
@@ -62,7 +62,13 @@ export class DDInstrumentedDatabase implements Database {
   ): Promise<DocumentDestroyResponse> {
     return tracer.trace("db.remove", span => {
       span?.addTags({ db_name: this.name, doc_id: id })
-      return this.db.remove(id, rev)
+      if (typeof id === "object") {
```

Comment:
We could extract the `id` and `rev` from the object, and to check if they are correct. Otherwise, passing an empty object will never complain